migrations/20161110175958_update_post_title_desc.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\Exception\UnsatisfiedDependencyException;
+
+class UpdatePostTitleDesc extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $pdo = $this->getAdapter()->getConnection();
+
+        $rows = $this->fetchAll(
+            "SELECT id
+                FROM form_stages
+                WHERE priority = 0"
+        );
+
+        $insert_title = $pdo->prepare(
+            "INSERT into
+                form_attributes
+                (`label`, `type`, `required`, `priority`, `cardinality`, `input`, `key`, `form_stage_id`)
+              VALUES
+                ('Title', 'title', 1, 0, 0, 'varchar', :key, :form_stage_id)"
+        );
+
+        $insert_desc = $pdo->prepare(
```

Comment:
This could be combined into a single insert with title.. just need to use `:title_key` and `:desc_key`